cpp/src/toppra/constraint.cpp

```diff
@@ -157,15 +176,17 @@ void LinearConstraint::allocateParams(std::size_t N,
 
     allocateLinearPart (N, k, m, constantF(), a, b, c, F, g);
   }
-  if (hasUbounds())
+  if (hasUbounds()) {
     ubound.resize(N);
-  if (hasXbounds())
+}
+  if (hasXbounds()) {
     xbound.resize(N);
 }
+}
 
 void LinearConstraint::computeParams(const GeometricPath& path, const Vector& gridpoints,
     Vectors& a, Vectors& b, Vectors& c, Matrices& F, Vectors& g,
-    Bounds ubound, Bounds& xbound)
+    const Bounds& ubound, Bounds& xbound)
```

Comment:
This should be `Bounds& ubound, Bounds& xbound)`

Reply:
Noted, will change.